Mention cloud dashboard and enhanced deployment instructions (#198)
infrawatch/documentation: doc-Service-Telemetry-Framework/modules/proc_setting-up-grafana-to-host-the-dashboard.adoc

grafana-deployment-7fc7848b56-sbkhv   1/1     Running   0          1m
----

> . Verify the grafana datasources installed correctly:

Reviewer: ```suggestion
. Verify the Grafana data sources installed correctly:
```

> . Verify the Grafana data sources installed correctly:

Reviewer 2: ```suggestion
. Verify that the Grafana data sources installed correctly:
```

+
[source,bash]
----
$ oc get grafanadatasources
----
+
----
NAME                    AGE
default-datasources     20h
----

> . Verify the grafana route exists:

Reviewer: ```suggestion
. Verify the Grafana route exists:
```

+
[source,bash]
----
$ oc get route grafana-route
----
+
----

> NAME            HOST/PORT                                                                   PATH   SERVICES          PORT   TERMINATION   WILDCARD
> grafana-route   grafana-route-service-telemetry.apps.ocp46stf.lab.upshift.rdu2.redhat.com          grafana-service   3000   edge          None

Reviewer: ```suggestion
NAME            HOST/PORT                                          PATH   SERVICES          PORT   TERMINATION   WILDCARD
grafana-route   grafana-route-service-telemetry.apps.infra.watch          grafana-service   3000   edge          None
```

----
grafanadashboard.integreatly.org/rhos-dashboard created
----
. Import the cloud dashboard:
+
[source,bash,options="nowrap"]
----
$ oc apply -f https://raw.githubusercontent.com/infrawatch/dashboards/master/deploy/rhos-cloud-dashboard.yaml
----
+
----
grafanadashboard.integreatly.org/rhos-cloud-dashboard created
----
[WARNING]

> Some panels in the cloud dashboard require the collect virt plugin hostname_format to be set to `name uuid hostname` in the stf-connectors.yaml. If not configured, affected dashboards will remain empty.

Reviewer: ```suggestion
Some panels in the cloud dashboard require the collectd virt plugin hostname_format to be set to `name uuid hostname` in the stf-connectors.yaml. If not configured, affected dashboards will remain empty.
```

Reviewer 2: @leifmadsen V minor edits to this, is the insertion of the word parameter correct?

> Some panels in the cloud dashboard require specific values be configured for the collectd virt plugin hostname_format parameter in the stf-connectors.yaml. If not configured some panels in the dashboard will be affected.

Reviewer 2: ```suggestion
Some panels in the cloud dashboard require specific values be configured for the collectd virt plugin hostname_format parameter in the stf-connectors.yaml. If not configured some panels in the dashboard will be affected.
```

The last sentence is a bit vague... how will the dashboards be affected if you don't chnage the settings?

Author: Updated

[source,yaml]
---
parameter_defaults:
    ExtraConfig:
        collectd::plugin::virt::hostname_format: name uuid hostname

> ---

Author: I intended that this configuration snippet be apart of the warning box. Not sure if the formatting I used renders this correctly.

Reviewer: Fixed this via https://asciidoctor.org/docs/asciidoc-writers-guide/#admonition-blocks. Accidentally committed the change instead of a suggest.

. Verify that the dashboards are available:
+
[source,bash]
----
$ oc get grafanadashboards
----
+
----
NAME                   AGE
rhos-dashboard         7d21h
rhos-cloud-dashboard   7d21h
----

. Retrieve the Grafana route address:
+
[source,bash]
----
$ oc get route grafana-route -ojsonpath='{.spec.host}' 
----
+
[source,bash,options="nowrap"]
----
grafana-route-service-telemetry.apps.infra.watch
----
+

> . Navigate to https://<GRAFANA-ROUTE-ADDRESS> in a web browser. Replace <GRAFANA-ROUTE-ADDRESS> with the value retrieved in the previous step.

Reviewer 2: ```suggestion
. Navigate to https://<GRAFANA-ROUTE-ADDRESS> in a web browser. Replace <GRAFANA-ROUTE-ADDRESS> with the value that you retrieved in the previous step.
```

----
grafanadashboard.integreatly.org/rhos-dashboard created
----
. Import the cloud dashboard:
+
[source,bash,options="nowrap"]
----
$ oc apply -f https://raw.githubusercontent.com/infrawatch/dashboards/master/deploy/rhos-cloud-dashboard.yaml
----
+
----
grafanadashboard.integreatly.org/rhos-cloud-dashboard created
----
[WARNING]

> Some panels in the cloud dashboard require the following configuration in the stf-connectors.yaml:

Reviewer: ```suggestion
Some panels in the cloud dashboard require specific values be configured for the collectd virt plugin hostname_format parameter in the stf-connectors.yaml. If not configured some panels in the dashboard will be affected.
```

[source,yaml]
---
parameter_defaults:
    ExtraConfig:
        collectd::plugin::virt::hostname_format: name uuid hostname
---

> If not added, no data will show in affected panels. 

Reviewer: ```suggestion
```

> . Verify that the dashboards installed correctly:

Reviewer: ```suggestion
. Verify that the dashboards are available:
```

+
[source,bash]
----
$ oc get grafanadashboards
----
+
----
NAME                   AGE
rhos-dashboard         7d21h
rhos-cloud-dashboard   7d21h
----

. Retrieve the Grafana route address:
+
[source,bash]
----
$ oc get route grafana-route -ojsonpath='{.spec.host}' 
----
+
[source,bash,options="nowrap"]
----

> grafana-route-service-telemetry.apps.ocp46stf.lab.upshift.rdu2.redhat.com

Reviewer: ```suggestion
grafana-route-service-telemetry.apps.infra.watch
```
